src/app/clusters/ecosystem-information-server/ecosystem-information-server.h

```diff
@@ -150,6 +150,24 @@ class EcosystemInformationServer
 public:
     static EcosystemInformationServer & Instance();
 
+    /**
+     * @brief Add EcosystemInformation Cluster to endpoint so we respond appropriately on endpoint
+     *
+     * EcosystemInformation cluster is only ever on dynamic bridge endpoint. If cluster is added
+     * to a new endpoint, but does not contain any ecosystem information presently,
+     * this is called to let ECOINFO cluster code know it is supposed to provide blank attribute
+     * information on this endpoint.
+     *
+     * This approach was intentionally taken instead of relying on emberAfDeviceTypeListFromEndpoint
```

Comment:
Are there unit tests for this cluster? Do they need to be updated to test this code?

Reply:
Unit test are getting after TE2. They do not exist, but this cluster code is written in a way that is currently testable and we don't want to break that paradime because I intend on adding tests when after TE2